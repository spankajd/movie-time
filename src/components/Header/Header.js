import './Header.scss';

const Header = () => {

    return <div className="header">
        <span className='title'>Movie Time</span>
    </div>
}

export default Header;