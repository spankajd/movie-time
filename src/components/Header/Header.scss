.header {
    width: 100%;
    padding: 20px 0;
    background: #13518e;
    text-align: center;

    .title {
        font-family: 'Poppins-SemiBold';
        color: #FFF;
        font-size: 36px;
    }

    @media (max-width: 768px) {
        padding: 15px 0;

        .title {
            font-size: 28px;
        }
    }
}