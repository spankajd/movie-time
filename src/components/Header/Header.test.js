// import React from "react";
// import { render } from "@testing-library/react";
// import Header from "./Header";

// test("renders Header component correctly", () => {
//   const { asFragment } = render(<Header />);
//   expect(asFragment()).toMatchSnapshot();
// });
import { render, screen, fireEvent } from "@testing-library/react";
import '@testing-library/jest-dom'; // Provides additional matchers
import Header from "./Header";

describe("Header Component", () => {
    it("renders Header", () => {
      render(<Header />);
      const headerElement = screen.getByText(/Movie Time/i);
      expect(headerElement).toBeInTheDocument();
    });
    it("matches snapshot", () => {
        const { asFragment } = render(<Header />);
        expect(asFragment()).toMatchSnapshot();
    });
});