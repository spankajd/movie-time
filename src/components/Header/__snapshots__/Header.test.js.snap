// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <span
      class="title"
    >
      Movie Time
    </span>
  </div>
</DocumentFragment>
`;
